WEB-INF/classes/gov/noaa/pfel/erddap/jte/Subscription.java

```diff
@@ -0,0 +1,32 @@
+package gov.noaa.pfel.erddap.jte;
+
+import gov.noaa.pfel.erddap.util.EDMessages.Message;
+import gov.noaa.pfel.erddap.util.EDStatic;
+import gov.noaa.pfel.erddap.util.Subscriptions;
+import java.text.MessageFormat;
+
+public class Subscription {
+  public static String getFormated(String value, String val) {
+    return MessageFormat.format(value, val);
+  }
+
+  public static String getMessage(Message message, int language) {
+    return EDStatic.messages.get(message, language);
+  }
+
+  public static String getAddHtml() {
```

Comment:
Not a big deal to leave like this, but I'm not sure why we don't just access `Subscriptions.VALIDATE_HTML` etc directly in the template. If we did that and moved the generic methods to `TemplateUtil`, this class could be removed entirely.